docs/content/preview/integrations/camunda.md

```diff
@@ -0,0 +1,173 @@
+- Double-click on the Start Event. A text box will open. Name the Start Event “Payment Retrieval Requested”.
+- Following it create a service event.
+  Click on the start event. From its context menu, select the activity shape (rounded rectangle). It will be placed automatically on the canvas, and you can drag it to your preferred position. Name it Charge Credit Card. Change the activity type to Service Task by clicking on the activity shape and using the wrench button.
+![alt text](https://docs.camunda.org/get-started/quick-start/img/modeler-step2.png)
+- Similarly add an End Event named Payment Received.
+![alt text](https://docs.camunda.org/get-started/quick-start/img/modeler-step3.png)
+- We are now required to configure the service task.
+![alt text](https://docs.camunda.org/get-started/quick-start/img/modeler-step5.png)
+  - First, configure an ID for the process. Type payment-retrieval in the property field Id. The property ID is used by the process engine as an identifier for the executable process, and it’s best practice to set it to a human-readable name.
+  - Second, configure the Name of the process. Type Payment Retrieval in the property field Name.
+  - Finally, make sure the box next to the Executable property is checked. If you don’t check this box, the process definition is ignored by the process engine.
+- Save the BPMN Diagram
+- Implement an external task worker
+After modeling the process, we want to execute some business logic.
+Camunda Platform is built so that your business logic can be implemented in different languages. You have the choice which language suits your project best. We are going to use JavaScript(NodeJs)
+- Create a new NodeJS project
+```bash
+mkdir charge-card-worker
+cd ./charge-card-worker
+npm init -y
+```
+- Add Camunda External Task Client JS library
+```bash
+npm install camunda-external-task-client-js
+npm install -D open
+```
+
+Implement the NodeJS script
+Next, we’ll create a new ExternalTaskClient that subscribes to the charge-card topic.
+
+When the process engine encounters a service task that’s configured to be externally handled, it creates an external task instance on which our handler will react. We use Long Polling in the ExternalTaskClient to make the communication more efficient.
+
+Next, you need to create a new JavaScript file, e.g. worker.js, that looks like the following:
+```js
+const { Client, logger } = require('camunda-external-task-client-js');
+const open = require('open');
+
+// configuration for the Client:
+//  - 'baseUrl': url to the Process Engine
+//  - 'logger': utility to automatically log important events
+//  - 'asyncResponseTimeout': long polling timeout (then a new request will be issued)
+const config = { baseUrl: 'http://localhost:8080/engine-rest', use: logger, asyncResponseTimeout: 10000 };
+
+// create a Client instance with custom configuration
+const client = new Client(config);
+
+// susbscribe to the topic: 'charge-card'
```

Comment:
I think we can remove the `open()` line here. Or at least we should point to something on OUR OWN site, instead of theirs.